internal/kafka/internal/presenters/kafka.go

```diff
@@ -23,7 +23,7 @@ func ConvertKafkaRequest(kafkaRequestPayload public.KafkaRequestPayload, dbKafka
 	kafka.Region = kafkaRequestPayload.Region
 	kafka.Name = kafkaRequestPayload.Name
 	kafka.CloudProvider = kafkaRequestPayload.CloudProvider
-	kafka.MultiAZ = kafkaRequestPayload.MultiAz
+	kafka.MultiAZ = kafkaRequestPayload.DeprecatedMultiAz
```

Comment:
This can be removed now but let's keep it and it will be removed when the multi_az field is removed from the request payload.